x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_suggestions.ts

```diff
@@ -186,14 +169,18 @@ function getSuggestionsForLayer(
   const sameStateSuggestions: Array<VisualizationSuggestion<State>> = [];
 
   // if current state is using the same data, suggest same chart with different presentational configuration
-
   if (seriesType !== 'line' && xValue.operation.scale === 'ordinal') {
```

Comment:
It's possible that we should remove the `flip` suggestion entirely, what do you think? It's unclear to me when this suggestion would be useful:

<img width="492" alt="Screenshot 2019-10-01 16 49 45" src="https://user-images.githubusercontent.com/666475/65999512-870e3680-e46b-11e9-92ce-901cc01a1fd7.png">